ui/src/app/components/with-new-user-satisfaction-survey-modal.spec.tsx

```diff
@@ -91,30 +85,39 @@ describe('withNewUserSatisfactionSurveyModal', () => {
     jest
       .spyOn(surveysApi(), 'validateOneTimeCodeForNewUserSatisfactionSurvey')
       .mockImplementationOnce(() => Promise.reject());
-
     expect(notificationStore.get()).toBeNull();
-
     await createWrapperAtPath(`?surveyCode=${code}`);
-
-    expect(notificationStore.get().title).toBeTruthy();
-    expect(notificationStore.get().message).toBeTruthy();
+    await waitFor(() => expect(notificationStore.get().title).toBeTruthy());
+    await waitFor(() => expect(notificationStore.get().message).toBeTruthy());
```

Comment:
I am not sure if this is the case or not, but if title and message attributes are being set at the same time, you might be able to get away on just the first statement having a waitfor. I don't think it is a big deal either way though.